### PR TITLE
Feature - upload files to specific folder on Azure File Share

### LINKED ISSUE #104: Azure File Share: Provide Script to upload files to specific folder in a File Share
**Is your feature request related to a problem? Please describe.**
Uploading files onto an Azure File Share can be automated if a script can be provided, which is part of the module 
**Arcus.Scripting.Storage.FileShare**

**Describe the solution you'd like**
Something similar to was is done here:
https://github.com/mbraekman/Powershell/blob/master/Azure/AzureStorage-UploadFilesToAzureFileShare.ps1

Suggested command name (open to better suggestions):
- _Copy-AzFileShareStorageFiles_

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by stijnmoreels**
Creates new PS module function to upload targeted files in a specific source folder to a destination folder on a Azure File Share.

Closes #104

### CODE REVIEW

File: docs/preview/features/powershell/azure-storage-fileshare.md [hunk 1]
```diff
@@ -30,7 +31,30 @@ Creates a new folder within the Azure File Share resource.
...
 ```powershell
-Create-AzFileShareStorageFolder -ResourceGroupName "shipping-resources" -StorageAccountName "tracking-account-storage" -FileShareName "returned" -FolderName "containers"
+PS> Create-AzFileShareStorageFolder -ResourceGroupName "shipping-resources" -StorageAccountName "tracking-account-storage" -FileShareName "returned" -FolderName "containers"
 # Creating 'containers' directory in file share..
 # Directory 'containers' has been created..
 ```
+
+## Copying files to a folder on an Azure file share
+
+Upload a series of files at a given folder, matching an optional file mask, to a Azure File Share.
```
Comment (mbraekman) [inline docs/preview/features/powershell/azure-storage-fileshare.md:41]:
```suggestion
Upload a set of files from a given folder, optionally matching a specific file mask, to an Azure File Share.
```

File: src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1 [hunk 2]
```diff
@@ -29,3 +29,43 @@ function Create-AzFileShareStorageFolder {
 }
 
 Export-ModuleMember -Function Create-AzFileShareStorageFolder
+
+<#
+ .Synopsis
+  Upload a series of files at a given folder to a Azure File Share.
```
Comment (mbraekman) [inline src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1:35]:
```suggestion
  Upload a set of files from a given folder to an Azure File Share.
```

File: src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1 [hunk 3]
```diff
@@ -29,3 +29,43 @@ function Create-AzFileShareStorageFolder {
...
+
+ .Description
+  Upload a series of files at a given folder, matching an optional file mask, to a Azure File Share.
```
Comment (mbraekman) [inline src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1:38]:
```suggestion
  Upload a set of files from a given folder, optionally matching a specific file mask, to an Azure File Share.
```

File: src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1 [hunk 4]
```diff
@@ -29,3 +29,43 @@ function Create-AzFileShareStorageFolder {
...
+
+ .Parameter ResourceGroupName
+  The resource group containing the Azure File Share.
+
+ .Parameter StorageAccountName
+  The Azure Storage account name that is hosting the Azure File Share.
```
Comment (mbraekman) [inline src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1:44]:
```suggestion
  The name of the Azure Storage account that is hosting the Azure File Share.
```

Just noticed that this was part of the Create-AzFileShareStorageFolder function as well. Could you update the description over there as well?

File: docs/preview/features/powershell/azure-storage-fileshare.md [hunk 5]
```diff
@@ -30,7 +31,30 @@ Creates a new folder within the Azure File Share resource.
...
+
+| Parameter               | Mandatory | Description                                                                                                            |
+| ----------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`     | yes       | The resource group containing the Azure File Share.                                                                    |
+| `StorageAccountName`    | yes       | The Azure Storage account name that is hosting the Azure File Share.                                                   |
```
Comment (mbraekman) [inline docs/preview/features/powershell/azure-storage-fileshare.md:46]:
```suggestion
| `StorageAccountName`    | yes       | The name of the Azure Storage account that is hosting the Azure File Share.                                                   |
```

File: src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1 (hunk 4 above)
Comment (stijnmoreels) [reply src/Arcus.Scripting.Storage.FileShare/Arcus.Scripting.Storage.FileShare.psm1]: Yes, done! Thanks for being thorough.

**Review by mbraekman — APPROVED** [review]

**Pull request merged**